css/style.css

```diff
@@ -0,0 +1,82 @@
+@media (max-width:700px){
+  #welcome-section{
+    color:red;
+  }
+}
+#welcome-section{
+  height : 100vh;
+  width: 100%;
+  background-color: green;
+}
+
+#welcome-section h1{
+  position:absolute;
+  left:50%;
+  top:50%;
+  bottom:0;
+  right:0;
+  /* transform: translate(-50%); */
```

Comment:
Good job so far, please try to avoid commenting the style attribute in your css file as it can be overwhelming and unmanageable.